ThirteenLtda/planning-ompl_geometric_planning
# check if the environment and robot paths were informed

This PR checks if the environment and robot paths were informed. If these paths are not informed, the OMPL will select the default environment.

How to test
---------------

Use the unit tests to check if this modification affected the previous code:

```bash
autoproj test enable planning/ompl_geometric_planning
acd planning/ompl_geometric_planning
amake
./build/test/planning_test
```
